python-base/ubuntu20.04-pyston2.2/docker/base-prepare.sh

```diff
@@ -34,6 +37,8 @@ chown -R "${USER}":"${GROUP}" /src
 bash /src/docker/scripts/configure_poetry.sh
 
 # Cleanup
+apt-get purge -y cargo
```

Comment:
Are the `-dev` libraries necessary after installation?

Reply:
Not 100% sure how they are used, will need to test and get back.

Reply:
A quick test shows that they don't seem to be required (tested by removing them and that I could build a working image with a small FastAPI application that runs nicely in the image).

Reply:
... so the PR is now updated with changes that removes those `*-dev` packages as well.